app/operations/shopify/variant_operations.rb

```diff
@@ -0,0 +1,22 @@
+module Shopify
+  class VariantOperations
```

Comment:
I think the questions that stemmed from the `ProductOperations` class fit here as well.